packages/js/src/shared-admin/components/premium-upsell-card.js

```diff
@@ -35,9 +35,10 @@ export const PremiumUpsellCard = ( { link, linkProps, promotions } ) => {
 
 		getPremium = createInterpolateElement(
 			sprintf(
-				/* translators: %1$s and %2$s expand to a span wrap to avoid linebreaks. %3$s expands to "Yoast SEO Premium". */
-				__( "%1$sAll annual plans incl. %2$s", "wordpress-seo" ),
+				/* translators: %1$s and %3$s expand to a span wrap to avoid linebreaks. %2$s expands to "Yoast SEO Premium". */
+				__( "%1$sBuy %2$s%3$s", "wordpress-seo" ),
 				"<nowrap>",
+				"Yoast SEO Premium",
 				"</nowrap>"
```

Comment:
```suggestion
				/* translators: %1$s and %2$s expand to a span wrap to avoid linebreaks. %3$s expands to "Yoast SEO Premium". */
				__( "%1$sBuy%2$s %3$s", "wordpress-seo" ),
				"<nowrap>",
				"</nowrap>",
				"Yoast SEO Premium"

```